feature-libs/cart/_index.scss

```diff
@@ -20,3 +19,9 @@ $useLatestStyles: true;
 @import './quick-order/index';
 @import './wish-list/index';
 @import './import-export/index';
+
+@import 'cxbase/blocks/modal';
+
+@import 'misc/hamburger';
+@import 'misc/dialog';
+@import 'misc/table';
```

Comment:
Same as previous. Let's include the package name

`@import '~@spartacus/styles/scss/misc/table';`

Reply:
What do you think about just putting this import in `feature-libs/cart/base/styles/components/_cart-item-list.scss` 

as that's the only place that is using extending .cx-agnostic-table

OR 

do you think it should be at the top level?